### PR TITLE
Update cmake to follow recent policy

### PR DESCRIPTION
**Pull request by jcarpent**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -44,25 +44,24 @@ include(cmake/eigen.cmake)
...
+IF(CMAKE_VERSION VERSION_GREATER "3.0.0")
+  CMAKE_POLICY(SET CMP0048 NEW)
+  IF(CMAKE_VERSION VERSION_GREATER "3.9.0")
+    PROJECT(${PROJECT_NAME} VERSION ${PROJECT_VERSION_FULL} LANGUAGES CXX DESCRIPTION ${PROJECT_DESCRIPTION})
+  ELSE(CMAKE_VERSION VERSION_GREATER "3.9.0")
+    PROJECT(${PROJECT_NAME} VERSION ${PROJECT_VERSION_FULL} LANGUAGES CXX)
+  ENDIF(CMAKE_VERSION VERSION_GREATER "3.9.0")
+ELSE(CMAKE_VERSION VERSION_GREATER "3.0.0")
+  PROJECT(${PROJECT_NAME} CXX)
+ENDIF(CMAKE_VERSION VERSION_GREATER "3.0.0")
```
Comment (jmirabel) [inline CMakeLists.txt]: I will wait for feedback in https://github.com/jrl-umi3218/jrl-cmakemodules/pull/285 before merging.

**Comment by jmirabel** [comment]
I update your PR according to change in jrl-umi3218/jrl-cmakemodules#285

File: CMakeLists.txt [hunk 2]
```diff
@@ -51,17 +51,8 @@ APPLY_DEFAULT_APPLE_CONFIGURATION()
 
 OPTION(BUILD_PYTHON_INTERFACE "Build the python bindings" OFF)
 
-# Tell CMake that we compute the PROJECT_VERSION manually.
-IF(CMAKE_VERSION VERSION_GREATER "3.0.0")
-  CMAKE_POLICY(SET CMP0048 NEW)
-  IF(CMAKE_VERSION VERSION_GREATER "3.9.0")
-    PROJECT(${PROJECT_NAME} VERSION ${PROJECT_VERSION_FULL} LANGUAGES CXX DESCRIPTION ${PROJECT_DESCRIPTION})
```
[the remark this answers is not in the document]
Comment (jmirabel) [reply CMakeLists.txt]: COMPUTE_PROJECT_ARGS should do the same as what you wrote. It should add the DESCRIPTION argument for CMake >= 3.9.0.

Do you have a different behavior ?

[the remark this answers is not in the document]
Comment (jcarpent) [reply CMakeLists.txt]: No, but you removed it from my commits. Could you add it back?

[the remark this answers is not in the document]
Comment (jcarpent) [reply CMakeLists.txt]: Before the release?! ;)

[the remark this answers is not in the document]
Comment (jmirabel) [reply CMakeLists.txt]: I *moved* it to there https://github.com/jrl-umi3218/jrl-cmakemodules/pull/285/commits/a3f6ce066b95b9cf707b36a9016182c42c1f05da

[the remark this answers is not in the document]
Comment (jcarpent) [reply CMakeLists.txt]: Got it. Thanks a lot!

**Pull request merged**